File: app/src/main/java/com/cappielloantonio/play/subsonic/models/Lyrics.java
package com.cappielloantonio.play.subsonic.models;

import com.tickaroo.tikxml.annotation.Attribute;
import com.tickaroo.tikxml.annotation.Xml;

@Xml
public class Lyrics {
    @Attribute(name = "value")
    protected String content;
    @Attribute
    protected String artist;
    @Attribute
    protected String title;

    public String getContent() {
        return content;
    }

    public void setContent(String value) {
        this.content = value;
    }

    public String getArtist() {
        return artist;
    }

    public void setArtist(String value) {
        this.artist = value;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String value) {
        this.title = value;
    }
}

File: app/src/main/java/com/cappielloantonio/play/subsonic/models/Lyrics.kt
package com.cappielloantonio.play.subsonic.models

import com.tickaroo.tikxml.annotation.Attribute
import com.tickaroo.tikxml.annotation.TextContent
import com.tickaroo.tikxml.annotation.Xml

@Xml(name = "lyrics")
class Lyrics {
    @TextContent
    var content: String? = null

    @Attribute
    var artist: String? = null

    @Attribute
    var title: String? = null
}